docs/style-guide.md

```diff
@@ -0,0 +1,80 @@
+# Project-Base Challenge Style Guide
+
+This guide aims to explain our overall approach to how project-based challenges (e.g. Step 1) are structured and written.
+
+## Instructional Content
+
+The instructions for a step challenge should be as concise as possible to inform the camper what to do, with the exception of steps that introduce a new concept. When introducing a new concept, you'll generally use the following format:
+
+```md
+Definition
+
+Explanation
+
+Instructions
+```
+
+As an example:
+
+```md
+The <dfn>`alt`</dfn> attribute provides alternative text to display if the image fails to load, or to be read to assistive technology.
+
+The `alt` attribute should provide the same information in text form that a user would get if they visually saw the image.
+
+Add an `alt` attribute to your image with the value `An orange cat playing with a string`.
+```
+
+Breaking the text up like this prevents a wall of text, and increases the chance the camper will read.
+
+## Tests
+
+The tests for each step should be as granular as possible.
+
+Rather than a test that says `You should create a new div element with an id set to main and a class set to container`, you should have three separate tests:
+
+- You should create a new div element.
+- Your new div element should have an `id` attribute set to `main`.
+- Your new div element should have a `class` attribute set to `container`.
+
+This provides clarity for where a camper should look at their code to correct the issue.
+
+### How to Test Code
+
+If you are testing HTML, most likely you will want to query the `document` directly. The `document` object in the test runner refers specifically to the embedded preview of the camper's code, so you can query the DOM as you would in a browser.
+
+If you are testing CSS, you will typically want to use the CSS Helper. The CSS Helper is a module that allows you to query the CSS that the camper has written. It is available in the tests by calling `new __helpers.CSSHelp(document)`. <!-- TODO: separate docs for curriculum helper-->.
```

Comment:
[collaboration request]: Can I create the curriculum helper docs with you? I can give a "fairly new coders" perspective to the guide. 😄 
```suggestion
If you are testing CSS, you will typically want to use the CSS Helper. The CSS Helper is a module that allows you to query the CSS that the camper has written. It is available in the tests by calling `new __helpers.CSSHelp(document)`. <!-- TODO: separate docs for curriculum helper-->.
```

Reply:
I'll be drafting these shortly, you'll be welcome to comment on the PR!